SPEC/FILES.md

```diff
@@ -722,15 +722,77 @@ Where:
   - `offset` is an Integer with the byte offset to begin reading from.
   - `count` is an Integer with the maximum number of bytes to read.
 
-`callback` must follow the `function (err, buf) {}` signature, where `err` is an Error if the operation was not successful and `buf` is a Buffer with the contents of `path`.
+`callback` must follow the `function (err, buf) {}` signature, where `err` is an Error if the operation was not successful and `buf` is a [`Buffer`][b] with the contents of `path`.
 
 If no `callback` is passed, a promise is returned.
 
 **Example:**
 
 ```JavaScript
 ipfs.files.read('/hello-world', (err, buf) => {
-  console.log(buf.toString())
+  console.log(buf.toString('utf8'))
+})
+
+// Hello, World!
+```
+
+#### `readReadableStream`
+
+> Read a file into a [`ReadableStream`][rs].
+
+##### `Go` **WIP**
+
+##### `JavaScript` - ipfs.files.readReadableStream(path, [options], [callback])
+
+Where:
+
+- `path` is the path of the object to read.
```

Comment:
Not an object, it is a file. (Objects in IPFS are single IPLD nodes)

Reply:
From what I can see the terms seem to be used interchangeably.  I've change references to 'objects' to 'file', let me know if I've missed any..

Reply:
Excellent, thank you!